pages/linux/pacman-files.md

```diff
@@ -0,0 +1,32 @@
+# pacman --files
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --files --help`
+
+- Update the package database:
+
+`sudo pacman --files --refresh`
+
+- Find the package that owns a specific file:
+
+`pacman --files {{filename}}`
```

Comment:
Is this is filename only, or does it support a `path/to/file`?

Reply:
as a user you can user the `filename` or the exact full path to file like `usr/bin/git` and the output if different:
```
$  pacman -F git
 extra/git 2.30.1-1 [installed]
    usr/bin/git
    usr/lib/git-core/git
    usr/share/bash-completion/completions/git
    usr/share/git/mw-to-git/bin-wrapper/git
 pacacommunity/drupal 9.0.6-2
    usr/share/webapps/drupal/core/tests/Drupal/Tests/Composer/Plugin/Scaffold/fixtures/scripts/disable-git-bin/git
$  pacman -F usr/bin/git
usr/bin/git is owned by extra/git 2.30.1-1
```

Reply:
I think it's better to keep it as "filename" 👍🏻